lib/foreman_deployments/engine.rb

```diff
@@ -46,6 +46,10 @@ class Engine < ::Rails::Engine
       Apipie.configuration.checksum_path += ['/foreman_deployments/api/']
     end
 
+    initializer "foreman_deployments.require_dynflow", :before => "foreman_tasks.initialize_dynflow" do |app|
+      ::ForemanTasks.dynflow.require!
```

Comment:
Not sure why but I get name error on `::ForemanTasks`. Explicit `require 'foreman_tasks'` fixes it.